lnbits/docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
       PROXY_AUTH_ADD: "false"
   
   web:
-    image: lnbitsdocker/lnbits-legend:0.9.4@sha256:0dbcc5ad5c5d4439c4ee5148206ecd3fbc28422aec2847a3f17e968205dab68b
+    image: lnbitsdocker/lnbits-legend:0.9.5.3@sha256:0dbcc5ad5c5d4439c4ee5148206ecd3fbc28422aec2847a3f17e968205dab68b
```

Comment:
The Docker image digest needs updating here. You can get the multi-arch digest by running: `docker pull <image>` and it will display the digest at the bottom.